src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -78,6 +78,15 @@ public LowLevelClient(string name, string ns, string description, string library
             SubClients = Array.Empty<LowLevelClient>();
         }
 
+        private static string GetClientName(string name)
+        {
+            if (Configuration.ModelNamespace && name == "Models")
+            {
+                return "ModelsOps"; // Avoid conflict with the Models namespace
```

Comment:
Better to give out log to indicate that we change the client name.

Reply:
we would do that through diagnostics.
Writing things to stdout/stderr is already causing issues.
I will add a TODO here to track when the system is ready, here we should report a warning diagnostic